test/addressService.test.ts

```diff
@@ -15,3 +44,201 @@ test("count: null request - expect error returned", async () => {
   expect(error).toBeDefined();
   expect(error).toEqual(NULL_ADDRESS_REQUEST_ERROR);
 });
+
+test("count: bad zipcode - expect count of 0", async () => {
+  await addressService
+    .count({ body: { zipcode: "12" } })
+    .then((response) => {
+      expect(response).toBeDefined();
+      expect(response.count).toEqual(0);
+    })
+    .catch((err) => {
+      fail("addressService.count - Unexpected Error: " + err);
+    });
+}, 90000);
+
+test("distance: null request - expect error returned", async () => {
+  let error: any;
+  await addressService
+    .distance(null)
+    .then(() => {
+      fail("addressService.distance: Unexpected Success");
+    })
+    .catch((err: Error) => {
+      error = err;
+    });
+
+  expect(error).toBeDefined();
+  expect(error.message).toEqual(NULL_ADDRESS_REQUEST_ERROR);
```

Comment:
expect assertions should be inside of the catch block. No need to put them out here in the very rare occurrence that moving outside of the block context causes race condition issues.